src/controllers/review.js

```diff
@@ -9,7 +9,9 @@ const create = (req, res) => {
       message: "The request body is empty",
     });
 
-  ReviewModel.create(req.body)
+  const review = req.body;
+  review.caretaker = req.userId;
```

Comment:
This is actually the owner Id and not the caretaker id because its the owner that will be issuing the api call.